engine/src/main/java/org/terasology/rendering/world/WorldRendererLwjgl.java

```diff
@@ -442,14 +406,14 @@ public void renderWorld(Camera camera) {
         DefaultRenderingProcess.getInstance().beginRenderLightGeometryStencilPass();
         Material program = Assets.getMaterial("engine:prog.simple");
```

Comment:
So, the Asset Manager generates at runtime a Material "engine:prog.simple" out of the shader "engine:simple". And the way to reference the material associated with a shader (rather than the shader itself) is to insert the "prog." string within the asset URI. Correct? 

Wouldn't this more appropriately handled via some call on the shader? I.e. shader.getMaterial() which in turn might generate one if one isn't available yet?